src/infrastructure/QueryParams.ts

```diff
@@ -34,21 +34,35 @@ export class QueryParams {
      * Page size between 10 and 100, otherwise 10
      */
     public pageSize = 10;
-    /**
-     * Id after which we want objects to be returned
-     */
-    public id?: string;
     /**
      * Order of transactions.
      * DESC. Newer to older.
      * ASC. Older to newer.
      */
+
     public order: Order = Order.DESC;
 
     /**
-     * Constructor
+     * Id after which we want objects to be returned
+     */
+    public id?: string;
+
+    /**
+     * Creates an instance of QueryParams.
+     * @param {{
+     *         pageSize?: number,
+     *         order?: Order,
+     *         id?: string;
+     *     }} configuration arguments
      */
-    constructor() {
+    constructor(args: {
```

Comment:
What about 
````
 constructor(args?: {
        pageSize?: number,
        order?: Order,
        id?: string;
    }) {
        if (args && args.pageSize) {
            this.setPageSize(args.pageSize);
        }
        if (args && args.order) {
            this.setOrder(Order[args.order]);
        }
        if (args && args.id) {
            this.setId(args.id);
        }
    }
````

So you can have a call like

`new QueryParams() `without the emtpy {}

Reply:
oops sorry, just updated